src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Ball randomBall = new Ball();
+        randomBall.setColor(new ColorSupplier().getRandomColor());
+        randomBall.setNumber(new Random().nextInt(100));
+        System.out.println(randomBall);
```

Comment:
Lottery has to give you random ball, it's not Lottery's responsibility to println()